openquake/hazardlib/tests/gsim/ita19/bindi_2011_backbone_delta_test.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+# vim: tabstop=4 shiftwidth=4 softtabstop=4
+#
+# Copyright (C) 2014-2018 GEM Foundation
+#
+# OpenQuake is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# OpenQuake is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with OpenQuake. If not, see <http://www.gnu.org/licenses/>.
+
+import numpy as np
+import unittest
+
+from openquake.hazardlib.imt import PGA, SA, PGV
+from openquake.hazardlib.gsim.ita19.bindi_2011_backbone import BindiEtAl2011Ita19Low
+
+
+class GetDeltaTest(unittest.TestCase):
```

Comment:
Could this just go in the same file as bindi_2011_backbone_test.py, just to avoid adding unnecessary files.